pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -161,21 +162,29 @@ type reconciler struct {
 	// idAllocator provides interfaces to allocate and release uint32 id.
 	idAllocator *idAllocator
 
-	// priorityAssigner provides interfaces to manage OF priorities.
-	priorityAssigner *priorityAssigner
+	// priorityAssigners provides interfaces to manage OF priorities for each OVS table.
+	priorityAssigners map[binding.TableIDType]*priorityAssigner
 
-	// priorityMutex prevents concurrent priority re-assignments
-	priorityMutex sync.RWMutex
+	// priorityMutex prevents concurrent priority re-assignments for each OVS table.
+	priorityMutex map[binding.TableIDType]*sync.RWMutex
```

Comment:
It should help to merge the two maps to one, to reduce lookup times?

Reply:
Done